src/components/inputs/Selector.tsx

```diff
@@ -33,6 +29,12 @@ export default function Selector({ getSelectedValue, startIcon, value, selectorT
 
   const setSelectorRef = useFocusableInput(isOpen)
 
+  function detectSelectorType(option: unknown) {
+    return selectorType === SelectorType.ASSIGNEE_SELECTOR
+      ? ((option as IAssigneeCombined).name as string) || ((option as IAssigneeCombined).givenName as string)
```

Comment:
Typecasting is done here to cast the `unknown` type for `option`. We can make use to typeguard as it is more safer approach and I think it is better to do this in a separate PR to avoid conflicts. This has to be changed in multiple files.